.ci/scripts/distribution/qa-testbench.sh

```diff
@@ -1,7 +1,50 @@
-#!/bin/sh -eux
+#!/bin/bash -eux
 
 chmod +x clients/go/cmd/zbctl/dist/zbctl
 
-alias zbctl="clients/go/cmd/zbctl/dist/zbctl"
+zbctl="clients/go/cmd/zbctl/dist/zbctl"
 
-zbctl create instance qa-protocol --variables "${QA_RUN_VARIABLES}"
+"${zbctl}" create instance external-tool-integration --variables "${QA_RUN_VARIABLES}"
+
+businessKey="${BUSINESS_KEY}"
+
+waiting=1
+
+echo "Waiting for result of $businessKey"
+
+while [ $waiting -eq 1 ]; do
+
+    "${zbctl}"  activate jobs "$businessKey" > activationresponse.txt 2>error.txt
```

Comment:
Is `error.txt` used for anything? Or do you just want to silence it? If that's the case, you can just redirect to /dev/null, i.e.

```suggestion
    "${zbctl}"  activate jobs "$businessKey" > activationresponse.txt 2>/dev/null
```